jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.py

```diff
@@ -34,6 +34,7 @@ class MetadataViewer(PluginTemplateMixin, DatasetSelectMixin):
     show_primary = Bool(False).tag(sync=True)
     has_comments = Bool(False).tag(sync=True)
     metadata = List([]).tag(sync=True)
+    metadata_filter = Unicode().tag(sync=True)

```

Comment:
since this is a plugin traitlet, _all_ instances of the plugin will be filtered simultaneously.  This is consistent with other behavior across the app, but we could consider pulling it out of the traitlet and keeping the search to the single instance down the road.